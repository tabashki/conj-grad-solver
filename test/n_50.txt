-6.3757196157e+00
5.5689169533e+00
-1.0665748132e+00
-3.4130902612e+00
-9.1785856127e-01
1.7987946233e-01
7.7827318645e-01
6.7817393164e+00
-9.6011271184e+00
1.0879047166e+01
-6.9845136700e+00
1.7356148531e+00
-3.8970382162e+00
3.2991049265e+00
1.7167308456e+00
-5.7150653696e+00
8.5241702617e+00
-3.5946571004e+00
-9.6501692514e+00
1.2279429342e+01
-5.3948561385e+00
7.1120372419e+00
-9.8122209724e+00
8.3471782382e+00
-3.9308288489e+00
-7.6769072650e+00
1.5722158303e+01
-1.2625481664e+01
3.2419373960e+00
5.9347930094e+00
-9.6630778488e+00
6.4396181351e+00
4.4486082531e-01
-8.6941546007e+00
9.1869057464e+00
-5.1039221096e+00
3.2638597001e+00
6.9564458281e-01
-2.2077249004e+00
3.4543419102e+00
-1.1841407565e+00
1.7388276459e-01
-8.3551183836e+00
7.9483752666e+00
2.3949881813e+00
-7.4264118379e+00
7.5652763252e+00
-4.2573532419e+00
-2.2547854763e+00
5.1835231139e+00
