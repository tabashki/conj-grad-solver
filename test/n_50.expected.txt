3.3759303811e+00
3.7614114662e-01
2.9452688654e+00
4.4478217710e+00
2.5372844154e+00
-2.9111150152e-01
-2.9396279561e+00
-4.8098712242e+00
1.0162482408e-01
-4.5880062460e+00
1.6014098494e+00
8.0631227492e-01
1.7468295535e+00
-1.2096913842e+00
-8.6710739536e-01
1.1922074391e+00
-2.4635430960e+00
2.4048766306e+00
3.6786392569e+00
-4.6977673682e+00
-7.9474465104e-01
-2.2865780724e+00
3.3336257481e+00
-8.5839140378e-01
3.2967696825e+00
3.5211019198e+00
-3.9314731078e+00
4.3381101672e+00
-1.7788221749e-02
-1.1317492146e+00
3.6890828019e+00
-1.1531630303e+00
4.4420927252e-01
2.4864424007e+00
-4.1654790718e+00
-1.6304947980e+00
-4.1994326338e+00
-3.5045107694e+00
-2.1139443223e+00
-2.9311027755e+00
-2.9391931853e-01
1.1591233820e+00
2.7860488470e+00
-3.9421440715e+00
-2.7219617235e+00
8.9320880590e-01
-2.9180325026e+00
8.3600251410e-01
3.3268428896e-01
-2.4254194125e+00
